po/perform_translations/translator.py

```diff
@@ -0,0 +1,97 @@
+import re
+import dl_translate as dlt
+import polib
+import os
+
+mt = dlt.TranslationModel()
+
+
+# Variables for translator to edit
+
+print(mt.get_lang_code_map()) 
+
+language = str(input("What is your desired language to translate to? Input the language code: ")) # change to desired language to convert TO
+Reportmsgid = str(input("What is your desired Report Msgid Bugs To? " ))
+POT_Creation_Date = str(input("What is your desired POT Creation Date? "))
```

Comment:
You don't need this, you'll fetch this from the POT.

Reply:
Fixed this @gcampax